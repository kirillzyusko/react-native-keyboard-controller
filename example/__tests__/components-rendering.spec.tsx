import React from 'react';
import { View } from 'react-native';
import {
  KeyboardControllerView,
  KeyboardProvider,
} from 'react-native-keyboard-controller';
import { render } from '@testing-library/react-native';

function KeyboardControllerViewTest() {
  return <KeyboardControllerView statusBarTranslucent />;
}

function KeyboardProviderTest() {
  return (
    <KeyboardProvider statusBarTranslucent>
      <View style={{ width: 20, height: 20, backgroundColor: 'black' }} />
    </KeyboardProvider>
  );
}

describe('components rendering', () => {
  it('should render `KeyboardControllerView`', () => {
    expect(render(<KeyboardControllerViewTest />)).toMatchSnapshot();
  });

  it('should render `KeyboardProvider`', () => {
    expect(render(<KeyboardProviderTest />)).toMatchSnapshot();
  });
});
