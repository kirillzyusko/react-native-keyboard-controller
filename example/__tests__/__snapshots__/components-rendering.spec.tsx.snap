// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components rendering should render \`KeyboardControllerView\` 1`] = `
<KeyboardControllerView
  statusBarTranslucent={true}
/>
`;

exports[`components rendering should render \`KeyboardProvider\` 1`] = `
<KeyboardProvider
  statusBarTranslucent={true}
>
  <View
    style={
      {
        "backgroundColor": "black",
        "height": 20,
        "width": 20,
      }
    }
  />
</KeyboardProvider>
`;
